alpha/apps/kaltura/modules/kmc/lib/kmcUtils.class.php

```diff
@@ -114,10 +114,10 @@ public static function getPartnersUiconfs($partner_id, $tag)
 		return $confs;
 	}
 
-	public static function getLiveAUiconf()
+	public static function getLiveUiconf($uiConfTag)
 	{
 		$c = new Criteria();
-		$tag = 'livea_player';
+		$tag = $uiConfTag;
```

Comment:
Use $uiConfTag as is why do you define another variable for it.

Reply:
I don't really know. didn't actually think about it